docs/source/how_to/use_oauth2.rst

```diff
@@ -147,9 +147,59 @@ However, if the response is not successful, you’ll receive an error response.
 Using the access token
 ----------------------
 
-.. note::
+Now that you have the access token, you can get the user data by sending a
+``GET`` request to ``/open_auth/get_user_data/`` with the access token in the
+header as follows:
+
+.. code-block:: json
+
+  {
+    "Authorization": "Bearer vqygrcouTuyAYHZLz3rGcZf5FpPd3K"
+  }
+
+where ``vqygrcouTuyAYHZLz3rGcZf5FpPd3K`` is the access token.
+
+If the access token is valid, then you will receive a 200 response
+with a dictionary containing the user's information based on the scopes
+of the client app, similar to this:
+
+.. code-block:: json
+
+    {
+       "userId": 1234,
+       "username": "19114099",
+       "person":{
+          "shortName":"dhruvkb",
+          "fullName":"Dhruv Kanti Bhanushali",
+          "roles":[
+             {
+                "role":"Student",
+                "activeStatus":"ActiveStatus.IS_ACTIVE"
+             },
+             {
+                "role":"Maintainer",
+                "activeStatus":"ActiveStatus.IS_ACTIVE"
+             }
+          ],
+          ...
+       },
+       "student":{
+            "enrolmentNumber": "19114099",
+             ...
+       },
+       "contactInformation":{
+            "instituteWebmailAddress": "xyz@iitr.ac.in"
+             ...
+       }
+    }
+
+However, if the access token is invalid, you will receive the following error response.
+
+.. code-block::
```

Comment:
In the entire method, this is the only where a potential error can be thrown

```
token = request.headers['Authorization'].replace('Bearer ', '')

        access_token = AccessToken.objects.get(token=token)
        application = access_token.application

        if application is None:
            return Response(
                data="Associated application not found",
                status=status.HTTP_404_NOT_FOUND
            )
```

Reply:
I'll mention it's a 404 error

Reply:
I changed it

Reply:
No, this code is just to prevent the case when there isn't any application attached to the access token which will never happen though. There is an authentication class in the starting of the view which causes a 403 error if the token is invalid, which is a meaningful error instead of a 404.

Reply:
Ohhh

Reply:
Where can I find the code for that?

Reply:
Nevermind

Reply:
Done 👍